extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -140,14 +140,15 @@ class AmpAccordion extends AMP.BaseElement {
         // for details.
       });
 
+      const isExpanded = section.hasAttribute('expanded');
       const header = sectionComponents[0];
       header.classList.add('i-amphtml-accordion-header');
       header.setAttribute('role', 'button');
       header.setAttribute('aria-controls', contentId);
-      header.setAttribute(
-        'aria-expanded',
-        section.hasAttribute('expanded').toString()
-      );
+      header.setAttribute('aria-expanded', isExpanded.toString());
```

Comment:
Let's start thinking how we'd manage the experiment here. Let's assume we have a function in this module such as `function isDisplayLockingEnabledForAccordion(win: Window):boolean`. It'd likely be a combination of `'renderSubtree' in Element.prototype` and our experiment.